src/events/keyboard.js

```diff
@@ -615,18 +626,21 @@ p5.prototype._onkeydown = function(e) {
  * </div>
  */
 p5.prototype._onkeyup = function(e) {
+  this._setProperty('isKeyPressed', false);
+  this._setProperty('keyIsPressed', false);
+  this._setProperty('_lastKeyCodePressed', this._keyCode);
   this._downKeys[e.which] = false;
 
-  if (!this._areDownKeys()) {
-    this._setProperty('isKeyPressed', false);
-    this._setProperty('keyIsPressed', false);
+  if (e.which === 91 || e.which === 93) { // Meta key codes
```

Comment:
On my mac, the left meta key generates code 224, so this branch never gets executed. Is there a reason for using `e.which` over checking if `e.key === 'Meta'`?

Reply:
no their wasn't any specific reason for using e.which, really sorry I have changed it to check for e.key === 'Meta' for consistency accross different platforms